qutebrowser/config/configcommands.py

```diff
@@ -250,6 +250,60 @@ def config_unset(self, option, temp=False):
         with self._handle_config_error():
             self._config.unset(option, save_yaml=not temp)
 
+    @cmdutils.register(instance='config-commands')
+    @cmdutils.argument('option', completion=configmodel.list_option)
+    def config_add_list(self, option, value, temp=False):
+        """Append a value to a config option that is a list.
+
+        This appends an option to a config setting that is a list.
+
+        Args:
+            option: The name of the option.
+            value: The value to append to the end of the dictionary.
+            temp: Don't touch autoconfig.yml.
+        """
+        opt = self._config.get_opt(option)
+        valid_list_types = (configtypes.List, configtypes.ListOrValue)
+        if not isinstance(opt.typ, valid_list_types):
+            raise cmdexc.CommandError(":config-add-list can only be used for "
+                                      "lists")
+
+        with self._handle_config_error():
+            option_value = self._config.get_mutable_obj(option)
+            option_value.append(value)
+            self._config.update_mutables(save_yaml=not temp)
+
+    @cmdutils.register(instance='config-commands')
+    @cmdutils.argument('option', completion=configmodel.dict_option)
+    def config_add_dict(self, option, key, value, temp=False, replace=False):
+        """Add a value at the key within the option specified.
+
+        This adds an element to a dictionary. --replace is needed to override
+        existing values.
+
+        Args:
+            option: The name of the option.
+            key: The key to use.
+            value: The value to place in the dictionary.
+            temp: Don't touch autoconfig.yml.
+            replace: Whether or not we should replace, default is not.
```

Comment:
This sounds a bit odd - what about "Replace existing values. By default, existing values are not overwritten."?